src/pages/index.js

```diff
@@ -1,15 +1,30 @@
 import Head from "next/head";
-import { Navbar } from "../components";
+import { useEffect, useState } from "react";
+import { AddItemForm, DateView, Meter, Navbar, TodoView } from "../components";
+import { useAppContext } from "../context/AppContext";
 
 export default function Home() {
+  const { darkMode } = useAppContext();
+  let [dark, setDark] = useState("light");
+  useEffect(() => {
+    setDark(darkMode);
+  }, [darkMode]);
```

Comment:
Why this extra state for the dark mode you can directly use the value from the context right?